src/views/Teams/TeamDetails.js

```diff
@@ -0,0 +1,30 @@
+import React, { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import TeamDetailsCard from '../../components/Teams/TeamDetailsCard';
+import { fetchTeamById } from '../../services/teams';
+
+export default function TeamDetails() {
+  const params = useParams();
+  const teamId = params.id;
```

Comment:
Here we can use destructuring to pull the { id } from useParams(), making this a one liner :)